yarn-audit-known-issues

```diff
@@ -0,0 +1,2 @@
+https://npmjs.com/advisories/1432
```

Comment:
Can you create a jira ticket to resolve this pls?

Reply:
Jira ticket created.